packages/thirdweb/src/transaction/read-contract.ts

```diff
@@ -111,7 +111,26 @@ export type ReadContractOptions<
  *
  * const { data, isLoading } = useReadContract({
  *   contract,
- *   method: "function tokenURI(uint256 tokenId) returns (string)"});
+ *   method: "function tokenURI(uint256 tokenId) returns (string)"}),
+ *   params: [1n],
+ * });
+ * ```
+ *
+ * ### Using `resolveMethod`
+ * ```ts
+ * import { getContract, resolveMethod } from "thirdweb";
+ * import { sepolia } from "thirdweb/chains";
+ * import { useReadContract } from "thirdweb/react";
+ *
+ * const contract = getContract({
+ *   client,
+ *   address: "0x...",
+ *   chain: sepolia,
+ * });
+ *
+ * const { data, isLoading } = useReadContract({
+ *   contract,
+ *   method: resolveMethod("tokenURI"),
```

Comment:
Does this work by itself without an ABI on the contract?

Reply:
yes! but we lose the type-safe feature. The result will be of type "unknown"